Sources/Foundation/NSPathUtilities.swift

```diff
@@ -21,7 +21,9 @@ let validPathSeps: [Character] = ["/"]
 #endif
 
 public func NSTemporaryDirectory() -> String {
-    FileManager.default.temporaryDirectory.path()
+    FileManager.default.temporaryDirectory.withUnsafeFileSystemRepresentation {
```

Comment:
This was necessary to ensure that paths with drive letters do not have a leading slash (the file system representation standardizes that)